packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -27,12 +27,17 @@ function filenameFromIndex(index: number): string {
     return index === 0 ? "" : index.toString(); // support old tools...
 }
 
+let currSeq: number = 1;
+
 async function* loadAllSequencedMessages(
     documentService?: IDocumentService,
     dir?: string,
-    files?: string[]) {
-    let lastSeq = 0;
+    files?: string[],
+    sequenceNumber: number = 0) {
+    let lastSeq = sequenceNumber;
+    currSeq = sequenceNumber;
```

Comment:
This will always override the initial value of currSeq. So I am unable to get why you initialized it with 1 and then overriding it here with 0.